<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>replaceDialog</class>
 <widget class="QDialog" name="replaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Match &amp;case</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>77</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="findNextButton">
      <property name="text">
       <string>&amp;Find Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceButton">
      <property name="text">
       <string>&amp;Replace</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceAllButton">
      <property name="text">
       <string>Replace &amp;All</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>7</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Fi&amp;nd What:</string>
      </property>
      <property name="buddy">
       <cstring>findLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="findLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Re&amp;place with:</string>
      </property>
      <property name="buddy">
       <cstring>replaceLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="replaceLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>replaceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findNextButton</sender>
   <signal>clicked()</signal>
   <receiver>replaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceButton</sender>
   <signal>clicked()</signal>
   <receiver>replaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceAllButton</sender>
   <signal>clicked()</signal>
   <receiver>replaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
